### PR TITLE
Database

### PR DESCRIPTION
**Pull request by polyccon**
*  Queries for insert/save and get data in all tables
*  Seed.sql file for prepopulating database with data for tests
*  Tests

### CODE REVIEW

File: src/model/queryGenerator.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    backgrounds: ['user_id', 'background'],
+    appointments: ['user_id', 'worker_preferences',
+      'appointment_preferences', 'parent_involved', 'email',
+      'mobile', 'telephone', 'contact_preference'],
+  };
+  const dollars =  sections[sectionName].map( (item, index) => {
+    return '$'+(index+1);
+  });
+
+  return 'INSERT INTO about_me (' + sections[sectionName].join(', ')+') VALUES ('+
```
Comment (dangerdak) [inline src/model/queryGenerator.js]: about_me is hardcoded

File: tests/database.test.js [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
 
 test('Insert user into database', (t) => {
   dbReset()
     .then(() => {
-      return addUser('jam', 'jam@gmail.com', 'password');
+      return addUser('jam', 'jam1@gmail.com', 'password');
+    })
+    .then((id) => {
+      t.equal(typeof id, 'number', 'Returns a number (the user id)');
+      return addUser('jon', 'jam1@gmail.com', 'password');
```
Comment (dangerdak) [inline tests/database.test.js]: I think this causes our travis build to fail. something to do with having to rollback a database transaction which errors maybe...
- https://stackoverflow.com/questions/10399727/psqlexception-current-transaction-is-aborted-commands-ignored-until-end-of-tra
- https://stackoverflow.com/questions/38558254/postgresql-catching-transaction-error-and-rollback

File: src/model/database/db_build.js [hunk 3]
```diff
@@ -16,14 +13,10 @@ const build = () => {
...
     });
-}
+};
 
-/* instanbul ignore if */
-if (require === module) {
-  build();
-}
 
+build();
```
Comment (rebecacalvoquintero) [inline src/model/database/db_build.js]: 👍

Comment (dangerdak) [reply src/model/database/db_build.js]: :)

**Pull request merged**